### ISSUE TITLE
Right-clicking anywhere in the Automaton 1.3.0b window at any time causes a crash

### ISSUE THREAD

**Issue by TorVestergaard** [metherul/Automaton#49]
The title says it all, really - I'm not sure what else to add, except a screenshot. This error reproduces in all windows during all stages of downloading and installation, as far as I can tell. I nuked my install-in-progress a few times by accident, until I learned to stop being jittery and leave it alone while it did its thing.

![image](https://user-images.githubusercontent.com/26014195/55683001-f383ff00-593a-11e9-8f5a-1374819daf29.png)

(Edit: Kudos on this amazing application, though - just installed US4 with premium Nexus and auto downloading, and the whole experience was super smooth. You've done a great thing for the Skyrim modding community with this.)

**Comment by TorVestergaard** [comment]
Just noticed the log file - if it helps, here's the full contents from a session where I open Automaton.exe and right-click in the window immediately: 

```
[13:47:12.1915696] [.ctor] 07/04/2019 13:47:12 Automaton Start
[13:47:12.3391868] [.ctor] Long path enabled. Continuing.
[13:47:12.3627140] [.ctor]    at MS.Internal.Xaml.Parser.GenericTypeNameParser.Callout_FoundName(String prefix, String name) Prefix 'materialDesign' does not map to a namespace.
[13:47:14.5371564] [.ctor]    at System.Windows.Window.DragMove() Can only call DragMove when primary mouse button is down.

```

**Comment by metherul** [comment]
Ha, know issue. Will be fixing tonight, I think. Thanks for the heads up!

**Comment by metherul** [comment]
Fixed in release 1.3.2.0b IIRC.
